core/federated/RTI/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.12)
-project(RTI VERSION 1.0.0 LANGUAGES C)
+project(RTI VERSION 0.2.0 LANGUAGES C)
```

Comment:
I propose we start out with version 0.2.0 instead of 0.1.0 because there are older versions of the RTI, but they just don't have semantic versioning. 0.2.0 indicates that this is not the initial version